website/src/panel-layout.tsx

```diff
@@ -52,6 +59,8 @@ export const PanelLayout = (p: {
     variables: { system: cherokeeRepresentation },
   })
 
+  const [commentsPanel, setCommentsPanel] = useState(false)
```

Comment:
I think for consistency with editing you might call this `isCommenting, setIsCommenting`